CMakeLists.txt

```diff
@@ -367,11 +377,17 @@ endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 
 # enables testing and creates Make check command
 enable_testing()
-set(CMAKE_CTEST_COMMAND ctest -V)
+set(CMAKE_CTEST_COMMAND ctest)
 add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND})
 
 add_subdirectory(src)
 
+# Now create a useable config.h
+configure_file(
```

Comment:
as some of the variables (`HAVE_GOOD_YASM_ELF64 ` for instance) used by the this configure_file are defined in `src/CMakeLists.txt`, maybe we can instead moving this `configure_file()` call down into `src/CMakeLists.txt`?